docs/docs/cmd/login.md

```diff
@@ -79,13 +79,13 @@ m365 login --authType password --userName user@contoso.com --password pass@word1
 Log in to Microsoft 365 using a PEM certificate
 
 ```sh
-m365 login --authType certificate --certificateFile /Users/user/dev/localhost.pem --thumbprint 47C4885736C624E90491F32B98855AA8A7562AF1
+m365 login --authType certificate --certificateFile /Users/user/dev/localhost.pem
```

Comment:
Let's keep the existing examples and add new ones to show both possibilities

Reply:
Ok. I'm a bit to eager to deprecate thumbprint as an option 😅